browser/main/Detail/SnippetNoteDetail.js

```diff
@@ -738,6 +738,8 @@ class SnippetNoteDetail extends React.Component {
             enableSmartPaste={config.editor.enableSmartPaste}
             hotkey={config.hotkey}
             autoDetect={autoDetect}
+            // dateISO8601={config.ui.dateFormatISO8601}
```

Comment:
Please remove the code instead of just comment it out. Thank you :+1:

Reply:
I'm sorry I forgot to remove those before committing, it's done.